pyccl/neutrinos.py

```diff
@@ -68,28 +67,35 @@ def Omeganuh2(a, *, m_nu, T_CMB=_Defaults.T_CMB, T_ncdm=_Defaults.T_ncdm):
     return OmNuh2
 
 
-@warn_api(pairs=[("OmNuh2", "Omega_nu_h2")])
-def nu_masses(*, Omega_nu_h2, mass_split, T_CMB=None):
+@warn_api(pairs=[("OmNuh2", "m_total")])
+def nu_masses(*, m_total, mass_split, T_CMB=None, is_Omega_nu_h2=True):
```

Comment:
Too convoluted to do it that way. The new signature should be 
```python
def nu_masses(*, Omega_nu_h2=None, mass_split, T_CMB=None, m_nu=None)
```
Then, inside the function, decide what we want to do based on which is not `None`. I don't like flags in functions except for very special cases.

Reply:
I thought you didn't like `None`s ;-).
Sure, push this, that's fine with me.

Reply:
I don't like `None` defaults when they can be substituted with the default. Generally, `None` isn't bad when there is no other option.